assets/js/modules/analytics-4/components/widgets/PopularProductsWidget.js

```diff
@@ -95,38 +92,49 @@ export default function PopularProductsWidget( props ) {
 			: undefined
 	);
 
-	if ( ! showWidget ) {
-		return <WidgetNull />;
-	}
-
 	const { rows = [] } = report || {};
 
-	const columns = [
-		{
-			field: 'dimensionValues',
-			Component: ( { fieldValue } ) => {
-				const [ title, url ] = fieldValue;
-				const permaLink = getFullURL( siteURL, url.value );
-
-				return (
-					<Link
-						href={ permaLink }
-						title={ title.value }
-						external
-						hideExternalIndicator
-					>
-						{ title.value }
-					</Link>
-				);
+	const columns = useInViewSelect( ( select ) => {
```

Comment:
Not sure this needs to be inViewSelect, does it? We generally use this for selectors that query reporting data or other side-effects.